lib/elixir/lib/code.ex

```diff
@@ -650,6 +650,10 @@ defmodule Code do
       when non-existing atoms are found by the tokenizer.
       Defaults to `false`.
 
+    * `:static_atom_encoder` - The static atom encoder function, see
+      the section below. The `:static_atom_encoder` option overrides
```

Comment:
Try to avoid pointing to just "the section below" since this quickly becomes outdated if we move documentation around. Use this instead:

```suggestion
      the ["The `:static_atom_encoder` function" section](module-the-static_atom_encoder-function)
      below. The `:static_atom_encoder` option overrides
```